streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -127,13 +127,31 @@ public void createTopics() throws Exception {
     }
 
     @Test
-    public void shouldBeAbleToRunWithEosEnabled() throws Exception {
-        runSimpleCopyTest(1, SINGLE_PARTITION_INPUT_TOPIC, null, SINGLE_PARTITION_OUTPUT_TOPIC, false);
+    public void shouldBeAbleToRunWithEosAlphaEnabled() throws Exception {
+        shouldBeAbleToRunWithEosEnabled(StreamsConfig.EXACTLY_ONCE);
```

Comment:
We duplicate each test for eos-alpha and eos-beta (I did not use parameterized tests, because I want to add one more upgrade test that does not need to be parameterized)

Reply:
Fail concern -- not sure. We could also add an `EosUpgradeIntegartionTest` as its own class?

Reply:
I think parameterize this class with `eos-alpha` and `eos-beta` is a better idea --- for some tests that do not rely on the flag and hence would be a duplicated one, we can move it to a separate non-parameterized class.